test/REQUIRE

```diff
@@ -0,0 +1,2 @@
+julia 0.3
+Compat 0.8.0
```

Comment:
Why is this `REQUIRE` file necessary?  Aren't these requirements already implied by the top-level `REQUIRE`?   (I thought you only need a `test/REQUIRE` if your tests require additional packages not needed for the package itself.)